src/madness/chem/znemo.cc

```diff
@@ -1189,17 +1189,30 @@ Znemo::canonicalize(std::vector<complex_function_3d>& amo,
 
 }
 
-
-void Znemo::save_orbitals(std::string suffix) const {
-	suffix="_"+suffix;
-	const real_function_3d& dia=diafac->factor();
-	for (size_t i=0; i<amo.size(); ++i) save(amo[i],"amo"+stringify(i)+suffix);
-	for (size_t i=0; i<bmo.size(); ++i) save(bmo[i],"bmo"+stringify(i)+suffix);
-	for (size_t i=0; i<amo.size(); ++i) save(madness::abs(amo[i]),"absamo"+stringify(i)+suffix);
-	for (size_t i=0; i<bmo.size(); ++i) save(madness::abs(bmo[i]),"absbmo"+stringify(i)+suffix);
-	for (size_t i=0; i<amo.size(); ++i) save(madness::abs(amo[i]*dia),"diaamo"+stringify(i)+suffix);
-	for (size_t i=0; i<bmo.size(); ++i) save(madness::abs(bmo[i]*dia),"diabmo"+stringify(i)+suffix);
-
+void Znemo::save_orbitals(std::string suffix) const
+{
+	suffix = "_" + suffix;
+	const real_function_3d &dia = diafac->factor();
```

Comment:
This is a very minor point, but is there a reason we don't use `auto` very much in this repository? The PR looks good to me otherwise.

Reply:
no reason, feel free to use as much as readability allows :)